server/fleet/app.go

```diff
@@ -480,16 +480,21 @@ type ListOptions struct {
 	// (varies depending on entity, eg. hostname, IP address for hosts).
 	// Handling for this parameter must be implemented separately for each type.
 	MatchQuery string `query:"query,optional"`
-
 	// After denotes the row to start from. This is meant to be used in conjunction with OrderKey
 	// If OrderKey is "id", it'll assume After is a number and will try to convert it.
 	After string `query:"after,optional"`
+	// Used to request the metadata of a query
+	IncludeMetadata bool
```

Comment:
Seems like this should be a method, not a field, right? (The caller of `ListActivities` is not using this field, it's used internally AFAICS, I still need to finish review.)

Reply:
You mean a method on `ListOptions` and an unexported field instead? I'm curious why you'd prefer that, it's just trading a field for a method AIUI, it would still be an exported method as this needs to be set in the datastore.

Note that the idea behind this field was to allow us to roll out the pagination metadata progressively (for now, only `ListActivities`), ultimately all "list-style" calls would have that pagination and the `IncludeMetadata` field or method would be removed.

Reply:
spoke with @lucasmrod and he mentioned that we could dismiss this comment.